cmd/bb_remote_asset/main.go

```diff
@@ -36,15 +34,17 @@ func main() {
 		log.Fatal("Usage: bb_remote_asset bb_remote_asset.jsonnet")
 	}
 	var config bb_remote_asset.ApplicationConfiguration
+	var lifecycleState *global.LifecycleState
+	var err error
 	if err := util.UnmarshalConfigurationFromFile(os.Args[1], &config); err != nil {
 		log.Fatalf("Failed to read configuration from %s: %s", os.Args[1], err)
 	}
-	if err := global.ApplyConfiguration(config.Global); err != nil {
+	if lifecycleState, err = global.ApplyConfiguration(config.Global); err != nil {
```

Comment:
Maybe nicer to turn this into a separate if, without the scoped assignment? That allows you to still use := for the variable declaration.